lib/drivers/medtronicDriver.js

```diff
@@ -481,19 +486,25 @@ module.exports = function (config) {
 
                     } else {
                       var decoded = decodeMessage(packet.parser, result.bytes);
```

Comment:
If `decodeMessage` is changed to throw, try here, otherwise a type check will be required.